src/runtime/manifest.ts

```diff
@@ -1257,45 +1256,28 @@ ${e.message}
     // we generate storage stubs that contain the relevant information.
     if (item.origin === 'storage') {
       return manifest.newStore({
-        type,
-        name,
-        id,
-        storageKey: item.source,
-        tags,
-        originalId,
-        claims,
-        description: item.description,
-        version: item.version,
-        origin: item.origin,
+        type, name, id, storageKey: item.source, tags,
```

Comment:
Personally I find this harder to read but that's a non-blocking opinion, just feedback in case this becomes common

Reply:
I agree that the expanded view is a bit easier, but I think with all but one of the args being keyword only, saving the vertical space is worth it (although I originally collapsed this when I was introducing another call to `newStore` for the inline case - so there'd be 3 calls in the function - but that ended up re-using the existing one at the end of the function).